src/components/Charts/ChartVaccinations.tsx

```diff
@@ -1,37 +1,245 @@
+    },
+    [showTooltip, dateScale, marginLeft],
+  );
+
+  const getXPosition = (d: Column) => dateScale(getDate(d)) || 0;
+  const getYPosition = (d: Column) => yScale(getY(d));
+  const dateTicks = getTimeAxisTicks(dateFrom, dateTo);
 
-const getTooltipContent = (valueY: number) => {
-  return {
-    subtitle: getMetricDefinition(Metric.VACCINATIONS).extendedMetricName,
-    body: `${getPointText(valueY)}`,
-    width: 'auto',
-  };
+  return (
+    <Styles.PositionRelative style={{ height }}>
+      <svg width={width} height={height}>
+        <Group key="chart-container" top={marginTop} left={marginLeft}>
+          <GridLines
+            width={innerWidth}
+            height={innerHeight}
+            dateScale={dateScale}
+            yScale={yScale}
+            numTicksRows={5}
+            xTickValues={dateTicks.map(date => date.valueOf())}
+          />
+          {/* Axes */}
+          <Styles.Axis>
+            <AxisLeft
+              scale={yScale}
+              numTicks={5}
+              tickFormat={(value: number) => formatPercent(value, 0)}
+            />
+            <AxisBottom
+              top={innerHeight}
+              scale={dateScale}
+              tickValues={dateTicks}
+              tickFormat={d => xTickFormat(d, false)}
+            />
+          </Styles.Axis>
+          <RectClipGroup width={innerWidth} height={innerHeight + 2}>
+            {seriesList.map(({ label, data, type, params }) => (
+              <ChartSeries
+                key={`series-chart-${label}`}
+                data={data}
+                x={getXPosition}
+                y={getYPosition}
+                type={type}
+                yMax={innerHeight}
+                barWidth={0}
+                barOpacity={1}
+                params={params}
+              />
+            ))}
+          </RectClipGroup>
+          {tooltipOpen && tooltipData && (
+            <DataHoverMarkers
+              x={getXPosition}
+              y={getYPosition}
+              yMax={innerHeight}
+              seriesList={seriesList}
+              date={tooltipData.date}
+            />
+          )}
+          <ChartOverlay
+            width={innerWidth}
+            height={innerHeight}
+            xScale={dateScale}
+            onMouseOver={onMouseOver}
+            onMouseOut={hideTooltip}
+          />
+        </Group>
+      </svg>
+      {tooltipOpen && tooltipData && (
+        <Fragment>
+          <VaccinesTooltip
+            left={p => getXPosition(p) + marginLeft}
+            top={p => getYPosition(p) + marginTop}
+            date={tooltipData.date}
+            seriesList={seriesList}
+            subtext={''}
+          />
+        </Fragment>
+      )}
+      {width > 0 && seriesList.length > 0 && <ScreenshotReady />}
+    </Styles.PositionRelative>
+  );
 };
 
-const ChartVaccinations = ({
-  columnData,
-  height = 400,
-}: {
-  columnData: Column[];
+const ChartVaccinationsAutosize: React.FC<{
+  seriesList?: Series[];
```

Comment:
Thoughts on making this not a list? Just more explicit if you pass in the series as either an interface:
```
interface VaccinationSeries {
  iniated?: Series
  completed?: Series
}
```
or their own arguments

Reply:
I'm trying to unify the data model of charts, the array of series works well for Explore and this chart, I prefer to keep it